src/pages/Markets/MarketsHeader.tsx

```diff
@@ -38,6 +38,21 @@ export const MarketsHeader = () => {
     history.push('/explore')
   }, [history])
 
+  const maybeCategory = useMemo(
+    () =>
+      history.location.pathname.match(/\/markets\/category\/(?<category>[\w]+)/)?.groups?.category,
```

Comment:
suggestion: history.location.pathname.split('/').pop() (just pop off the last path param)

Reply:
nevermind, this is short sighted. all routes have a final part to the path lol. disregard.